receiver/filelogreceiver/README.md

```diff
@@ -153,4 +153,43 @@ The above configuration will read logs from the "simple.log" file. Some examples
 2023-06-20 12:50:00 DEBUG This is a test debug message
 ```
 
+## Fault tolerance, recovery and proper offset tracking
 
+File offset tracking is quite crucial to ensure the quality of this receiver.
+Filelog receiver has no native support for file offset tracking however this can be achieved by using
+the proper `extentions`.
+
+Filelog receiver's persistence can be covered by the usage of the following:
+- [filestorage](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage) extension,
+to ensure that Collector's restarts do not affect the log collection and offset tracking.
+- [exporterhelper persistent-queue](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#persistent-queue),
+to ensure that Collector's restarts do not affect the delivery of the already collected logs.
```

Comment:
```suggestion
```

I don't think it makes sense for receiver documentation to make recommendations about downstream configuration.

What do you think about adding a full collector config example in [`/examples`](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/examples) and linking to it from here?